### DISCUSSION TITLE
What is the meaning of this logo in search bar?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1526] [Q&A] [answered]
![IMG_20211030_084947](https://user-images.githubusercontent.com/79367916/139517978-d296632a-f6c7-431d-b8d5-863447cbf7cc.jpg)

**Comment by uazo** [comment, marked as answer]
read #788 
if you can provide [this](https://github.com/bromite/bromite/issues/788#issuecomment-784202210) it might help us
